packages/x-date-pickers-pro/src/internal/hooks/useStaticRangePicker/useStaticRangePicker.tsx

```diff
@@ -80,7 +80,12 @@ export const useStaticRangePicker = <
           {...slotProps?.layout}
           slots={slots}
           slotProps={slotPropsForLayout}
-          sx={sx}
+          sx={[
+            ...(Array.isArray(sx) ? sx : [sx]),
+            ...(slotProps?.layout?.sx && Array.isArray(slotProps?.layout?.sx)
```

Comment:
```suggestion
            ...Array.isArray(slotProps?.layout?.sx)
```

The following should be enough

Reply:
Yes but the typescript seems to not understand that  `Array.isArray(slotProps?.layout?.sx)` implies that `slotProps.layout.sx` is not undefined, so it complains the next line

Reply:
`Array.isArray(slotProps?.layout?.sx!) then